### PR TITLE
Kafkasource doc fix

### PR DESCRIPTION
**Pull request by Shashankft9**
Fixes #issue-number or description of the problem the PR solves
https://github.com/knative/docs/issues/2520

## Proposed Changes 

- updating couple of sections in the README.md of kafka/source since the logs captured have changed in the new releases.

### CODE REVIEW

File: docs/eventing/samples/kafka/source/README.md [hunk 1]
```diff
@@ -142,14 +142,41 @@ Tutorial on how to build and deploy a `KafkaSource` [Eventing source](../../../s
...
+           "nameKey": "logger",
+           "callerKey": "caller",
+           "messageKey": "msg",
+           "stacktraceKey": "stacktrace",
+           "lineEnding": "",
+           "levelEncoder": "",
+           "timeEncoder": "iso8601",
+           "durationEncoder": "",
+           "callerEncoder": ""
+         }
```
Comment (mattmoor) [inline docs/eventing/samples/kafka/source/README.md:169]: Remove trailing whitespace:
```suggestion
         }
```

File: docs/eventing/samples/kafka/source/README.md [hunk 2]
```diff
@@ -142,14 +142,41 @@ Tutorial on how to build and deploy a `KafkaSource` [Eventing source](../../../s
...
+           "callerKey": "caller",
+           "messageKey": "msg",
+           "stacktraceKey": "stacktrace",
+           "lineEnding": "",
+           "levelEncoder": "",
+           "timeEncoder": "iso8601",
+           "durationEncoder": "",
+           "callerEncoder": ""
+         }  
+       }
```
Comment (mattmoor) [inline docs/eventing/samples/kafka/source/README.md:170]: Remove trailing whitespace:
```suggestion
       }
```

File: docs/eventing/samples/kafka/source/README.md [hunk 3]
```diff
@@ -142,14 +142,41 @@ Tutorial on how to build and deploy a `KafkaSource` [Eventing source](../../../s
...
+           "lineEnding": "",
+           "levelEncoder": "",
+           "timeEncoder": "iso8601",
+           "durationEncoder": "",
+           "callerEncoder": ""
+         }  
+       } 
+
+   ```
+   Now manually delete the kafkasource deployment and allow the `kafka-controller-manager` deployment running in `native-sources` namespace to redploy it. Debug level logs should be visible now.
```
Comment (mattmoor) [inline docs/eventing/samples/kafka/source/README.md:173]: Remove trailing whitespace:
```suggestion
   Now manually delete the kafkasource deployment and allow the `kafka-controller-manager` deployment running in `native-sources` namespace to redploy it. Debug level logs should be visible now.
```

**Comment by Shashankft9** [comment]
@googlebot I signed it!

**Comment by Shashankft9** [comment]
Reopening another one since the username got mixed up here.

**Pull request closed without merging**